MetaMask/metamask-extension
# fix/BannerTip to TS

## Explanation

- Fixes #19119 

## Screenshots/Screencaps

### Before

https://github.com/MetaMask/metamask-extension/assets/79097544/e479481b-5b61-47a3-96cf-110f82b0f51b

### After

https://github.com/MetaMask/metamask-extension/assets/79097544/71ca80b4-ed09-4b99-93eb-195b9f671b59

unit tests
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/1849038a-d956-47fa-8e8d-3fe933802d63)

## Manual Testing Steps

- Check storybook controls for `BannerTip`
- Ensure unit tests pass by running `yarn jest ui/components/component-library/banner-tip`

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
